l1-contracts/contracts/zksync/libraries/BLS.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+library BLS {
```

Comment:
Please document the curve name something like "This library implements the BLS signing scheme over the BN254 curve" or [that](https://github.com/getwax/bls-wallet/blob/ce7f958fb74b10e2b83bb886638b4ec0f64d9fac/contracts/contracts/lib/hubble-contracts/contracts/libs/BLS.sol#L10)